### DISCUSSION TITLE
Add software to hello

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#350] [General]
Add software to hello OS
Choose from more office opportunities

- Zotero - Reference management
- Darktable - Photography processing application
-Notepad++ - Plain text editor
- VLC - Media player
- Mixxx - DJ mixing software
- Ardour - Open source music software
- Telegram - Communication software
- Signal - Communication software
- Transmission - BitTorrent client
- OpenShot - Video editor 
- FileZilla - FTP client

**Comment by probonopd** [comment]
* Only Qt based applications shall be considered. I think Darktable is Gtk, and hence will not look and feel native
* Telegram and Signal require a mobile phone nunmber and hence are not privacy friendly
* Ardour requires Jack and is hence not user friendly
* We already have a curated video editor: Shotcut (I think it has better performance?)
* We already have a curated video player: mpv (I think it is much smaller and has better performance?)
* Who still uses FTP?

Need to test:
* Mixxx
* Zotero

**Reply by louies0623** [reply]
We have some rural primary schools still using FTP...
For Darktable, shouldn't hello be more creator friendly?

**Reply by grahamperrin** [reply]
> Telegram and Signal require a mobile phone nunmber and hence are not privacy friendly

[Signal secure messaging can now identify you without a phone number – Naked Security](https://nakedsecurity.sophos.com/2020/05/22/signal-secure-messaging-can-now-identify-you-without-a-phone-number/)

**Reply by probonopd** [reply]
Article explicitly says:

> The disappointing news here, at least in our opinion, is that Signal isn’t yet announcing a way to use its product without handing over a phone number at all.

**Comment by kettle-7** [comment]
There must be a better way to install the apps than filling the system up with shortcuts to apps that chances are not everyone wants there.

This takes us back to the appshop idea that was abandoned and replaced with a temporary fix but said temporary fix doesn't do well as a user facing app for non power users.

**Reply by probonopd** [reply]
The unneeded placeholder apps can be deleted very easily.

**Reply by probonopd** [reply]
> This takes us back to the appshop idea

There is a small proof-of-concept called "Download Applications" in "Under Construction".

**Reply by louies0623** [reply]
maybe it's like mate boutique software, don't want the full store program to maintain, but also keep some software options, maybe this is the best Appshop compromise.

**Comment by louies0623** [comment]
Hey @kettle-7 If he is thinking of downloading the internal code of the program, can you design an UI for him?  (You can refer to mate boutique)

**Comment by louies0623** [comment]
Also requires an antivirus such as :[ClamAV](https://www.clamav.net/)
The same photography software as darktable : [rawtherapee](https://www.rawtherapee.com/)
Free and Open Source Game Engines for Developing Games : 
[Godot](https://godotengine.org/)
[amethyst](https://amethyst.rs/)
[panda3d](https://www.panda3d.org/download/sdk-1-10-9/)
[love2d](https://love2d.org/)
[libgdx](https://libgdx.com/)

**Comment by louies0623** [comment]
> A desktop system for creators that focuses on simplicity, elegance, and usability.

Maybe you can see their lives through videos, and you will know that the built-in software in the system is not enough.

- [Photographer](https://youtu.be/tUe5pxtbMt0)
- [Game development](https://youtu.be/z-baoW2-B-A)
- [Animation Studio](https://youtu.be/mC9jNhWfFQo)
